src/pages/messaging/agent-chat-api/v3.4/index.mdx

```diff
@@ -1373,13 +1381,14 @@ Transfers a chat to an agent or a group. The transferred chat must be active. Wh
 
 #### Request
 
-| Parameter     | Required | Data ype | Notes                                                                                                                                  |
-| ------------- | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------- |
-| `id`          | Yes      | `string` | Chat Id                                                                                                                                |
-| `target`      | No       | `object` | If missing, the chat will be transferred within the current group.                                                                     |
-| `target.type` | Yes      | `string` | `group` or `agent`                                                                                                                     |
-| `target.ids`  | Yes      | `array`  | `group` or `agent` IDs array                                                                                                           |
-| `force`       | No       | `bool`   | If `true`, always transfers active chats. Otherwise, fails when unable to assign any agent from the requested groups; default `false`. |
+| Parameter                    | Required | Data ype | Notes                                                                                                                                  |
+| ---------------------------- | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                         | Yes      | `string` | Chat ID                                                                                                                                |
+| `target`                     | No       | `object` | If missing, the chat will be transferred within the current group.                                                                     |
+| `target.type`                | Yes      | `string` | `group` or `agent`                                                                                                                     |
+| `target.ids`                 | Yes      | `array`  | `group` or `agent` IDs array                                                                                                           |
+| `ignore_agents_availability` | No       | `bool`   | If `true`, always transfers active chats. Otherwise, fails when unable to assign any agent from the requested groups; default `false`. |
```

Comment:
"always" is not valid anymore, I guess.

Reply:
ahh, I forgot to copy the updated text from RTM, nice catch